detection-rules/attachment_any_html_unsolicited.yml

```diff
@@ -7,7 +7,7 @@ references:
   - "https://ired.team/offensive-security/defense-evasion/file-smuggling-with-html-and-javascript"
   - "https://sandbox.sublimesecurity.com?id=106315e9-166a-4e0f-946e-88ff6fd5f9fd"
 type: "rule"
-severity: "medium"
+severity: "low"
```

Comment:
should the "new sender" version of this rule also be `low`?

Reply:
from my exp "new sender" is more likely to be a TP than this one